File: 0001_0100/0008/0008.py
with open("input.txt", "r") as f, open("output.txt", "w") as q:
    s = [int(x) for x in f.read().split()]
    if s[0] * s[1] == s[2]:
        q.write("YES")
    else:
        q.write("NO")

File: 0001_0100/0008/0008.go
package main

import (
	"fmt"
	"os"
)

func main() {
	f, _ := os.Open("input.txt")
	q, _ := os.Create("output.txt")
	var a, b, c int
	fmt.Fscan(f, &a, &b, &c)
	if a*b == c {
		fmt.Fprint(q, "YES")
	} else {
		fmt.Fprint(q, "NO")
	}
}
